torch/testing/_internal/common_methods_invocations.py

```diff
@@ -785,7 +786,8 @@ def sample_inputs_fliplr_flipud(op_info, device, dtype, requires_grad):
     OpInfo('addmm',
            dtypes=floating_types(),
            dtypesIfCPU=all_types_and_complex_and(torch.float16, torch.bfloat16),
-           dtypesIfCUDA=floating_types_and(torch.float16, torch.complex64, torch.complex128),
+           dtypesIfCUDA=floating_types_and(torch.float16, torch.complex64, torch.complex128,
+                                           *[torch.bfloat16] if tf32_is_not_fp32() else []),
```

Comment:
This will need a comment explaining the condition. Are we trying to test for Ampere or whether TF32 is enabled?